CppCoreGuidelines.md

```diff
@@ -6539,6 +6540,40 @@ That tends to work better than "cleverness" for non-specialists.
 
 * Flag throwing `hash`es.
 
+### <a name="Rc-memset"></a>C.90: Rely on constructors and assignment operators, not `memset` and `memcpy`
+
+##### Reason
+
+The standard C++ mechanism to construct a an instance of a type is to call its constructor. As specified in [C.41: A constructor should create a fully initialized object](#Rc-complete). No additional initialization, such as by `memcpy`, should be required.
```

Comment:
"construct a an instance"

The second sentence, "As specified in C.41: ..." doesn't seem like a complete sentence.

Reply:
Fixed & clarified.